themes/edx.org/lms/templates/header.html

```diff
@@ -104,60 +105,64 @@ <h1 class="hd logo-header">
                     self.real_user = getattr(user, 'real_user', user)
                     is_on_profile_page = data and data.get('profile_user_id') is not None
                   %>
-                  <li class="nav-item mt-2 nav-item-open-collapsed">
+                  <li class="nav-item nav-item-open-collapsed">
                     <a class="${'active ' if is_on_profile_page else ''}tab-nav-link" href="${reverse('learner_profile', args=[self.real_user.username])}">
                       ${_("Profile")}
                     </a>
                   </li>
-              % endif
-                </ul>
-          % endif
-        % endif
-        <ul class="navbar-nav navbar-right">
-          % if should_display_shopping_cart_func(): # see shoppingcart.context_processor.user_has_cart_context_processor
-            <a role="button" class="nav-item-open-collapsed btn-shopping-cart btn btn-secondary mr-3" href="${reverse('shoppingcart.views.show_cart')}">
-              <span class="icon fa fa-shopping-cart" aria-hidden="true"></span> ${_("Shopping Cart")}
-            </a>
+                % endif
+              </ul>
+            % endif
           % endif
-          <li class="nav-item mt-2 nav-item-open-collapsed">
-            <a href="${get_online_help_info(online_help_token)['doc_url']}"
-             target="_blank"
-             class="nav-link">${_("Help")}</a>
-          </li>
-          <%include file="user_dropdown.html"/>
+          <ul class="nav align-items-center">
+            % if should_display_shopping_cart_func(): # see shoppingcart.context_processor.user_has_cart_context_processor
+              <li>
+                <a role="button" class="nav-item-open-collapsed btn-shopping-cart btn btn-secondary mr-3" href="${reverse('shoppingcart.views.show_cart')}">
```

Comment:
Why does this have the role of button?

Reply:
I didn't write this code (I think I just indented it). I'll remove the role.